md2html.sh

```diff
@@ -16,7 +16,22 @@ while getopts "hl" opt; do
 done
 
 for file in "$@"; do
-	pandoc -f markdown_github "${file}" -t asciidoc -o "${file%.md}.tmp.adoc"
+        md2adoc_tool=""
+        # RHEL 9 in brew cannot use pandoc, hence trying kramdoc first
+        if type -p kramdown >/dev/null; then
+          md2adoc_tool=kramdown
+        elif type -p pandoc >/dev/null; then
+          md2adoc_tool=pandoc
+        fi
+        if [ $md2adoc_tool == kramdown ]; then
+          kramdoc --format=GFM --output="${file%.md}.tmp.adoc" "${file}"
```

Comment:
No Fedora package contains `kramdoc`, so this will be problematic to use on Fedora/RHEL.